contracts/protocol/PositionManager.sol

```diff
@@ -231,15 +232,15 @@ contract PositionManager is
             require(
                 int128(_pip) >=
                 (int256(getUnderlyingPriceInPip()) -
-                int128(maxFindingWordsIndex * 250)), Errors.VL_MUST_CLOSE_TO_INDEX_PRICE_LONG
+                int128(maxWordRangeForLimitOrder * 250)), Errors.VL_MUST_CLOSE_TO_INDEX_PRICE_LONG
```

Comment:
The minimum pip should be 1, in case `(int256(getUnderlyingPriceInPip()) - int128(maxWordRangeForLimitOrder * 250) < 0`